crates/wasm_module/src/opcodes.rs

```diff
@@ -183,6 +183,12 @@ pub enum OpCode {
     F64REINTERPRETI64 = 0xbf,
 }
 
+impl From<u8> for OpCode {
+    fn from(x: u8) -> Self {
+        unsafe { std::mem::transmute(x) }
+    }
```

Comment:
it would be so nice if we had something safe for this. if an invalid value ever makes it in here that is instant UB

Reply:
OK the only way I could think to do it was another massive `match`!
I did it with copy/paste and multi-line editing so I know it's all consistent. I didn't do any macro magic for that, because it felt like overkill. This file is very rarely touched.